### ISSUE TITLE
No events being logged 

### ISSUE THREAD

**Issue by krasno** [pyalarmdotcom/alarmdotcom#230]
Not sure when this started, but not seeing any events for my alarm system being logged.  Doors opening or closing, panel arming or disarming, basically nothing.  Tried restarting HA and did not solve.  Removed the integration and the package and reinstalled and same result.  No error messages and everything looks fine on the surface, but nothing is happening. I don't know if this is due to the new two factor authentication or not, but I have yet to set that up. Just using the same username and password method.

**Comment by elahd** [comment]
Dupe of #229. Would you mind enabling debug mode as described in that issue?

**Comment by sspaul1976** [comment]
Thank you for your response.  I ran the debug dump but did not see anything
out of the ordinary or was erroring. I did notice that everything seems to
pick up fine as long as the refresh occurs while the alarm system sensor is
active.  Meaning, if I leave a door open for over minute, when the
integration does it's refresh after 30 seconds, it will log the event of
both opening then closing.  If I open and close the door in normal use, it
is not picking it up.  Let me know if you still want the dump file.
